com.microsoft.mrtk.core/Utilities/DisplayType.cs

```diff
@@ -18,6 +18,11 @@ public enum DisplayType
         /// <summary>
         /// A display that blocks viewing of the physical world.
         /// </summary>
-        Opaque = 1
+        Opaque = 1,
+
+        /// <summary>
+        /// An unknown display type.
+        /// </summary>
+        Unknown = 2,
```

Comment:
Typically the "unknown", "none" enumeration values are `0`, so to catch uninitialized variables.

It probably doesn't make much difference here, but still it's probably a good practice.  What do you think about changing this to `Unknown = 0`?

I guess there is a concern about breaking users of the old value, but such breaking changes should be expected in a prerelease version.

Reply:
Updated